packages/release-brancher/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "release-brancher",
+  "version": "1.0.0",
+  "description": "Cut release branches",
+  "bin": "./build/src/bin/release-brancher.js",
+  "private": true,
```

Comment:
A recent concern - I worry a little about private modules that share a name (or could have a shared name) with something on the public npm registry.  The attack I'm specifically worried about is someone noticing this module name, and publishing a real package to the registry named `release-brancher`, which does Bad Stuff™️.  If someone forgets to `npm link` before `npx release-brancher`, they are getting the malicious code.  I completely defer to @bcoe's expertise on this topic, but this feels like a fun attack vector.

Reply:
I would actually like to distribute this to npm for partner teams to be able to install/run more easily.

Reply:
AH! In that case, you need to drop the `private: true` bit, that will actively prevent it from getting published to npm.

Reply:
Setting up this repo for multi-module releases will be done separately with @bcoe